docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+</td>
+<td>Admin API
+</td>
+<td>Kafka API
+</td>
+<td>Pandaproxy API
+</td>
+</tr>
+<tr>
+<td>Node
+</td>
+<td>
+
+`<cluster>-admin-api-node`
+
+</td>
+<td>
+
+`<cluster>-redpanda`
+
+</td>
+<td>
+
+`<cluster>-proxy-api-node`
+
+</td>
+</tr>
+<tr>
+<td>Client
+</td>
+<td>
+
+`<cluster>-admin-api-client`
+
+</td>
+<td>
+
+`<cluster>-user-client`
+
+</td>
+<td>
+
+`<cluster>-proxy-api-client`
+
+</td>
+</tr>
+</table>
+
+Each Certificate resource results in the creation of a secret resource of type `kubernetes.io/tls` that contains three components: `tls.crt`, `tls.key`, and `ca.crt`. 
+
+You can check them by running:
+
+```bash
+kubectl describe secret <cluster>-admin-user-client
+```
+
+The result should look like this: 
+
+```
+Name:  <cluster>-admin-user-client
+..
+Type:  kubernetes.io/tls
+Data
+====
+tls.key:  .. bytes
+tls.crt:  .. bytes
+ca.crt:   .. bytes
+```
+
+If you want to check their content you can run:
+
+```bash
+kubectl get secret <cluster>-admin-user-client -o yaml
+```
+
+`ca.crt` is provided when using a self-signed Certificate Authority (CA), which is the default case but may not be provided as part of the above secret if using a well-known authority.
+
+
+### Updating/ Renewing existing certificates
+
+The renewing process is done automatically by cert-manager. You can check [how they do it on their website](https://cert-manager.io/docs/usage/certificate/#renewal).
+
+The certificate duration is set by the operator to 5 years.
+
+When this process happens, Redpanda reads the new certificates without needing to restart. 
+
+
+## Using the certificates
+
+Now that your server is secured let's see how things work in the perspective of the client. 
+
+Each certificate has a Subject Alternative Name (SAN) that is set to the Fully Qualified Domain Name (FQDN) of the pods with and without a wildcard prefix:
+
+```
+X509v3 Subject Alternative Name: 
+    DNS:   <cluster>.default.svc.cluster.local,
+    DNS: *.<cluster>.default.svc.cluster.local
+```
+
+The non-wildcard DNS is typically used for bootstrapping. The wildcarded name lets you access instances directly.
+
+The above assumes that the external connectivity is disabled. When external connectivity is enabled and a subdomain (https://vectorized.io/docs/kubernetes-connectivity/#Using-names-instead-of-external-IPs) is provided the certificates are produced for external use. For example, the SAN entries would become `*.<subdomain>` and `<subdomain>`.
+
+Depending on where the client is located in relation to the Kubernetes cluster running Redpanda, the client certificate secrets may be mounted to local pods or be exported and used directly (as files) inside or outside of the Kubernetes cluster, as long as the Kafka API is exposed to the external network.
+
+There are multiple scenarios for consuming the certificates from a client application. For example, a client can be in the same Kubernetes cluster, can be in the same namespace, can have a requirement for mTLS, etc.
```

Comment:
```suggestion
There are multiple scenarios for consuming the certificates from a client application. For example, a client can be in the same Kubernetes cluster, can be in the same namespace, and can have a requirement for mTLS.
```
Etc is really unclear and it's best to avoid it if we can. You say "for example" so you don't need to list everything.